distributed/scheduler.py

```diff
@@ -1550,57 +1524,59 @@ def replicate(self, stream=None, keys=None, n=None, workers=None,
                     self.add_keys(address=w, keys=list(gathers[w]))
 
     def workers_to_close(self, memory_ratio=2):
```

Comment:
Unrelated to the changes in this PR, but I feel a docstring and/or explanation at top would make this method more understandable.